examples/plasticity/WellboreProblem_PoroDruckerPrager.xml

```diff
@@ -0,0 +1,286 @@
+<!--
+Problem: wellbore problem subjected to mixed mode loading and anisotropic far-field stresses
+Created by: Jian Huang
```

Comment:
Suggest removing the Created By and Last Updated lines.  We discourage people putting authorship lines and dates in the repo, as they will soon be out of date.  It is better to track contributions and changes via the Git Blame feature.

Reply:
Agree, will remove these lines.